### PR TITLE
Rsu effective terrain roughness height

### PR DESCRIPTION
**Pull request by j3r3m1**

### CODE REVIEW

File: geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy [hunk 1]
```diff
@@ -526,3 +526,97 @@ static IProcess rsuRoofAreaDistribution() {
...
+ *
+ * @return outputTableName Table name in which the rsu id and their corresponding indicator value are stored
+ *
+ * @author Jérémy Bernard
+ */
+static IProcess rsuEffectiveTerrainRoughnessHeight() {
+    return processFactory.create(
+            "RSU effective terrain roughness height",
+            [rsuTable: String, projectedFacadeAreaName: String, geometricMeanBuildingHeightName: String,
+             prefixName: String, listLayersBottom: Double[], numberOfDirection: Integer,
```
Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: double

Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: int

Comment (j3r3m1) [reply geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: The numberOfDirection must be an Integer. It is consistent with the rsuProjectedFacadeAreaDistribution IProcess

Comment (j3r3m1) [reply geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: I do not get it, int or double ?

Comment (ebocher) [reply geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: Sorry I'm pointing listLayersBottom

**Comment by ebocher** [comment]
You are wonderful!

**Pull request merged**